examples/react/getting-started/src/App.css

```diff
@@ -113,3 +107,50 @@ em {
   height: 100%;
   justify-content: space-between;
 }
+
+.ais-TrendingItems-item h2,
+.ais-RelatedProducts-item h2 {
+  display: -webkit-box;
```

Comment:
where did this style come from? seems copy-pasted from somewhere with autoprefixer maybe?

Reply:
Same styles as the ones I used for InstantSearch.js. They're the recommended ones for line clipping.